SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -0,0 +1,120 @@
+# [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling)
```

Comment:
We need to have this link in text, IMHO and to have "Unreal documentation" in brackets as part of the link - this is our convention to help orient users.

Reply:
Actually we can call out the Unreal docs:
"Before reading this topic, make sure you are familiar with the Unreal documentation on [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling).)